.github/workflows/pytest-gpu.yml

```diff
@@ -70,11 +70,15 @@ jobs:
       with:
         command: |
           #! /bin/bash
-          # Ensure clang and cuda libraries are visible
+          # Ensure clang, PGI and cuda libraries are visible
```

Comment:
nitpicking: Split the comment. First a comment for clang exports, then a comment for pgcc exports, then a comment for cuda libs exports

Reply:
Fixed